src/components/Collapse.jsx

```diff
@@ -60,16 +58,6 @@ const Collapse = ({
           style={{ display: isOpen ? 'block' : 'none' }}
           onClick={onOverlayClick}
         ></div>
-        <div className="close-div">
-          <button
-            className="btn close-menu"
-            type="button"
-            title={intl.formatMessage(messages.CloseMenu)}
-            onClick={onOverlayClick}
-          >
-            <Icon color="white" icon="it-close-big" padding={false} />
-          </button>
-        </div>
```

Comment:
Invece di rimuovere da qui il bottone e spostare la sua gestione all'interno dei figli, penso tu faccia meglio a spostare il bottone sotto a `{children}` per metterlo in fondo al contenuto